config/webpack/webpack.common.ts

```diff
@@ -25,14 +30,14 @@ const includeModules = [
     'expo-av',
 ].join('|');
 
-const envToLogoSuffixMap = {
+const envToLogoSuffixMap: Record<string, string> = {
     production: '',
     staging: '-stg',
     dev: '-dev',
     adhoc: '-adhoc',
 };
 
-function mapEnvToLogoSuffix(envFile) {
+function mapEnvToLogoSuffix(envFile: string): string {
```

Comment:
NAB but `envFile` => `environmentFile`. Let's do this for all the truncated variable names in these files (like `env` below too)